### DISCUSSION TITLE
Add WPForms support

### DISCUSSION THREAD

**Discussion by rvdsteege** [pronamic/wp-pronamic-pay#198] [Ideas]
Add support for WPForms.

- https://wpforms.com

From customer:

> Ik heb altijd een formulier van paytium aan mijn betaallink gekoppeld. Maar deze werkt sinds een paar dagen niet meer. Kan ik jullie link ook koppelen aan wpforms?

_Internal support ticket: https://secure.helpscout.net/conversation/1613803570/22620/_

**Comment by LeoOosterloo** [comment]
Interessante data over market share van deze plugin in vergelijking met Gravity Forms: https://www.slintel.com/tech/wordpress-plugins/gravityforms-vs-wpforms

GF is de grootste, daarna komt WPForms. Waar WPForms vooral groter is in Europa, Azië, Afrika en Zuid Amerika.

**Comment by LeoOosterloo** [comment]
WPForms Lite seems to have supported payment forms via Stripe, PayPal, and Square, among others, for some time now. Perhaps we can tie into this with Pronamic Pay. Based on market share of [Slintel.com](https://www.slintel.com/tech/wordpress-plugins/gravityforms-vs-wpforms), WPForms appears to be used more than Gravity Forms, particularly in Europe.
